modules/ROOT/pages/faq/index.adoc

```diff
@@ -1,6 +1,14 @@
 FAQ
 ===
 
+[[I want to upgrade from Community Version to Enterprise Version. What are the changes?]]
+== I want to upgrade from Community Version to Enterprise Version. What are the changes?
+
+In ownCloud Enterprise you will get access to new apps and features, mainly targeted towards enterprises.  Apps and features that ensure security for example.
+
+Your existing *files*, *shares* and *users* remain as they are. You can upgrade to the Enterprise version without concerns.
```

Comment:
I feel that it reads better when the two sentences are swapped and slightly reworked, as follows:

You can upgrade to the Enterprise version without concern, as your existing *files*, *shares* and *users* remain as they are.

Reply:
Don't you think that the sentence as it is puts the emphasis on the files shares and users, as they are in the first part of the sentence. In you rewrite it reads better, but the files shares and users are in the last part of the sentence, making the user read.